access.lua

```diff
@@ -208,17 +208,18 @@ end
 if is_logged_in then
     assets = {
                    ["/ynh_portal.js"] = "js/ynh_portal.js",
+                   ["/ynh_userinfo.json"] = "ynh_userinfo.json",
```

Comment:
What's the purpose of this file and this line?

Reply:
/ynh_userinfo.json is a file listed in several place of ssowat code (before this PR) (sometimes it's also called ynhpanel.json). Here i just make things working.

The goal is to return a json with data you could use to build a view. For example it contains the list of apps tiles the user can see. In my use case (reviving menu_ynh concept) i could use this to filter/disable menu item.